l2geth/eth/gasprice/rollup_gasprice.go

```diff
@@ -0,0 +1,35 @@
+package gasprice
+
+import (
+	"context"
+	"math/big"
+)
+
+type RollupOracle struct {
```

Comment:
I think we should make access to this struct safe for concurrency because it will be read from goroutines spawned on incoming RPC requests and it will be written to in a goroutine spawned by the `SyncService`. We can use a `sync.RWMutex`